.github/actions/issue-validator/src/repro-link.mjs

```diff
@@ -22,8 +22,7 @@ async function run() {
 
   // https://github.com/vercel/next.js/blob/canary/.github/ISSUE_TEMPLATE/1.bug_report.yml?plain=1
 
-  const start =
-    '### Link to the code that reproduces this issue or a replay of the bug'
+  const start = '### Link to the code that reproduces this issue'
   const end = '### To Reproduce'
   const linkRe = new RegExp(`${start}(.*)${end}`, 'is')
```

Comment:
```suggestion
  const linkRe = new RegExp(`${start}(?:or a replay of the bug)(.*)${end}`, 'is')
```

I think you'll need to make this backwards compatible with existing issues, right?

Reply:
No, this action only runs on newly created issues.